src/ctelnet.h

```diff
@@ -153,6 +154,13 @@ const int MTTS_STD_MNES = 512; // Client supports the Mud New Environment Standa
 const int MTTS_STD_MSLP = 1024; // Client supports the Mud Server Link Protocol for clickable link handling.
 const int MTTS_STD_SSL = 2048; // Client supports SSL for data encryption, preferably TLS 1.3 or higher.
 
+// https://tintin.mudhalla.net/protocols/mnes/
+const char MNES_IS = 0;
+const char MNES_SEND = 1;
+const char MNES_INFO = 2;
+const char MNES_VAR = 0;
+const char MNES_VAL = 1;
```

Comment:
These should be:
```cpp
const char NES_IS = 0;
const char NES_SEND = 1;
const char NES_INFO = 2;
const char NES_VAR = 0;
const char NES_VALUE = 1;
const char NES_ESC = 2;
const char NES_USERVAL = 3;
```
if we were doing a proper implementation of Telnet suboption **39** "New Environment."

Reply:
If you look in the original RFC you can see that the `NES_ESC`  is needed if the value to be passed *contains* any of the above values - and that if the value is `0xff`/255 then it has to be doubled to escape it from being seen as an `<IAC>` - as that does not happen automatically IIRC if it is sent via `socketOutRaw(...)`

Reply:
Implemented QString escapeNewEnviron(const QString&) to meet these requirements.

Reply:
Renamed and added the missing command codes.